modules.d/35network-manager/nm-config.sh

```diff
@@ -6,4 +6,11 @@ if [ -n "$netroot" ] || [ -e /tmp/net.ifaces ]; then
     echo rd.neednet >> /etc/cmdline.d/35-neednet.conf
 fi
 
+if getargbool 0 rd.debug -d -y rdinitdebug -d -y rdnetdebug; then
+    (
+        echo '[logging]'
+        echo 'level=TRACE'
+    ) > /run/NetworkManager/conf.d/initrd-logging.conf
```

Comment:
Does the directory `/run/NetworkManager/conf.d` exist already?

Reply:
fixed